go/oasis-net-runner/fixtures/default.go

```diff
@@ -221,6 +231,49 @@ func newDefaultFixture() (*oasis.NetworkFixture, error) {
 				fixture.ComputeWorkers[j].Runtimes = append(fixture.ComputeWorkers[j].Runtimes, rtIndex)
 			}
 			fixture.Clients[0].Runtimes = append(fixture.Clients[0].Runtimes, rtIndex)
+
+			// Runtime state paths to use to initialize the runtime with.
+			runtimeStatePath := runtimeStatePaths[i]
+			// Set workers runtime state.
+			for j := range fixture.ComputeWorkers {
+				fixture.ComputeWorkers[j].RuntimeStatePaths = append(fixture.ComputeWorkers[j].RuntimeStatePaths, runtimeStatePath)
+			}
+			if runtimeStatePath == "" {
+				continue
+			}
+
+			dbPath := filepath.Join(runtimeStatePath, database.DBFileBadgerDB)
+			_, err := os.Stat(dbPath)
+			if err != nil {
+				return nil, fmt.Errorf("runtime state path: %w", err)
+			}
+			db, err := badger.New(&mkvsAPI.Config{
+				DB:        dbPath,
+				Namespace: runtimeIDs[i],
+			})
+			if err != nil {
+				return nil, fmt.Errorf("opening state path: %w", err)
+			}
+			dbVersion, exists := db.GetLatestVersion()
+			if !exists {
+				return nil, fmt.Errorf("no version found in runtime state db: %v", dbVersion)
+			}
+			rts, err := db.GetRootsForVersion(context.Background(), dbVersion)
+			if err != nil {
+				return nil, err
+			}
+
+			var stateRootHash hash.Hash
+			for _, r := range rts {
+				if r.Type == storage.RootTypeState {
+					stateRootHash = r.Hash
+				}
+			}
```

Comment:
I would put this into a helper function (e.g. something like `getLatestStateRoot(path)`) to make this more clear.